tests/e2e/specs/pacbio_run_create.js

```diff
@@ -37,53 +68,48 @@ describe('Pacbio Run Create view', () => {
       .select('Do Not Generate')
       .get('#bindingKitBoxBarcode')
       .type('12345')
-      .get('button')
-      .contains('Update')
```

Comment:
Generally with e2e tests its better to use things like content, as it mimics the way the user looks at the page. It can get a bit tricky if you have multiple buttons with the same label, but then at least it forces you to work out how a user may distinguish between them.